src/todosRepository.js

```diff
@@ -0,0 +1,16 @@
+import * as ko from 'knockout';
+import { Todo } from './todo.model.js';
+
+export const todosRepository = {
+    load() {
+        console.log(window.localStorage);
+        let list = JSON.parse(localStorage.getItem('todoList'));
```

Comment:
It's better to explicitly say that localStorage is a part of window (readability of code, avoiding of possible errors) by using it through window.localStorage everywhere